root/edit/details/MergeAreas.js

```diff
@@ -16,7 +16,7 @@ type Props = {
 const MergeAreas = ({edit}: Props): React$Element<'table'> => (
   <table className="details merge-areas">
     <tr>
-      <th>{addColonText(lp('Merge', 'merge X into Y heading'))}</th>
+      <th>{addColonText(lp('Merge', 'verb, header'))}</th>
```

Comment:
My point with "'merge X into Y heading'" is that this is not a standalone line, but it's part of a set, `"Merge:" X "into:" Y`. As such, the two probably need to be translated in a way where they make sense together. But maybe I'm overthinking this?

Reply:
Good point, that wasn’t clear to me from commit <https://github.com/metabrainz/musicbrainz-server/pull/3068/commits/938fbe31c1c33f0b29af6e2b4dfb2ab74d19bcc3>.

Would the following suggestion work for you instead?

```suggestion
      <th>{addColonText(lp('Merge', 'verb, header, paired with Into'))}</th>
```

And then add the context `header, paired with Merge` to `Into` a few lines under?

Reply:
Following this suggestion in https://github.com/metabrainz/musicbrainz-server/pull/3083/commits/80bf5723aa2ee45ba88b253c886faf454ecfbaf8 in case it works for you.